### DISCUSSION TITLE
This error is often reported when calling the wallet in the mobile browser

### DISCUSSION THREAD

**Discussion by gwz-365461760** [WalletConnect/walletconnect-monorepo#5519] [Developer Support]
error:1,
WebSocket connection to 'wss://relay.walletconnect.org/?auth=eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJkaWQ6a2V5Ono2TWtuajlEMnhiejJjVHZiTVV4bjd6MWFQTERoZWpFSlViUlJNeVc5Zmp6WEFlWSIsInN1YiI6IjQ4MTI3MTk5MDc4NTNlODc2OWEyY2FmNmRmMzFiMjJjYTU1NjZlY2RhM2UxN2UwZmFkYmZjMjUxZGI1ZDkxZTkiLCJhdWQiOiJ3c3M6Ly9yZWxheS53YWxsZXRjb25uZWN0Lm9yZyIsImlhdCI6MTczMjU5MDAxNSwiZXhwIjoxNzMyNjc2NDE1fQ.7ybFzd-ZToVvcdhfuU6VKLAl2f1tXIiDIJrB6JUU7MNOSf0yiyMKNvvFyGUyasMqsq2zlgC31u3gtey_0nOJAQ&projectId=c186acb08d52967ac63d31236e08b458&ua=wc-2%2Fjs-2.17.0%2Fios-ios-18.1.1%2Fbrowser%3Ashop.soccern.io&useOnCloseEvent=true' failed: The network connection was lost.

error:2,
[Error] WebSocket connection to 'wss://relay.walletconnect.org/?auth=eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJkaWQ6a2V5Ono2TWtuajlEMnhiejJjVHZiTVV4bjd6MWFQTERoZWpFSlViUlJNeVc5Zmp6WEFlWSIsInN1YiI6Ijc1NWMyZTllYTQyOGE1MzA4N2E3OTFiZjg2NjIxNWFhYjQ4YjllMmZmMmYwOGQ0MWRmNjljYTg5YjRkMWZjN2MiLCJhdWQiOiJ3c3M6Ly9yZWxheS53YWxsZXRjb25uZWN0Lm9yZyIsImlhdCI6MTczMjU4OTI5MywiZXhwIjoxNzMyNjc1NjkzfQ.6ROG7drcybKMxgejl_eEvMN0nOw9yEoQDt6DoLEhCI8aIU1p4yWLKRQ0SbUnmGXU_PFT7Z5FpVA4y6aFoZeVCg&projectId=c186acb08d52967ac63d31236e08b458&ua=wc-2%2Fjs-2.17.0%2Fios-ios-18.1.1%2Fbrowser%3Ashop.soccern.io&useOnCloseEvent=true' failed: The network connection was lost.
[Error] Unhandled Promise Rejection: Error: No matching key. session topic doesn't exist: 92a5d57c175fe4f5d10283a353e8ae89d310058b23f29580dc6df7e462d415d5
	（匿名函数） (app.69fcff9b.js:6187:735015)
[Error] Failed to load resource: the server responded with a status of 503 () (balance, line 0)
[Error] Failed to load resource: the server responded with a status of 503 () (balance, line 0)

**Comment by danielsimao** [comment]
Having the same issue
